boards/st/stm32f411e_disco/stm32f411e_disco_stm32f411xe_B.overlay

```diff
@@ -29,3 +30,8 @@
 			  <&gpioe 5 GPIO_ACTIVE_HIGH>;
 	};
 };
+
+/* The B revision has the l3gd20 instead of i3g4250d, but no test board is available. */
```

Comment:
Non blocking: explanation could be clearer. Do you mean that you're not willing to add the node because you're  not able to test it ?